client/live.js

```diff
@@ -1,8 +1,40 @@
 var $ = require("jquery");
-var io = require("socket.io-client");
+var SockJS = require("sockjs-client");
 var stripAnsi = require('strip-ansi');
 require("./style.css");
 
+var recInterval = null;
+var sock = null;
+var hot = false;
+var currentHash = "";
+
+var newConnection = function(handlers) {
+	sock = new SockJS('/sockjs-node');
+
+	clearInterval(recInterval);
+
+	sock.onclose = function() {
+		handlers.close();
+
+		// Try to reconnect.
+		sock = null;
+		recInterval = setInterval(function () {
```

Comment:
Since you are clearing the interval immediately on execution, why not just:

    recTimeout = setTimeout(newConnection, 2000, handlers);

Since newConnection is only called once, the clearInterval is not needed. Same for the other thing in live.js.